src/twisted/internet/asyncioreactor.py

```diff
@@ -37,14 +37,18 @@ class AsyncioSelectorReactor(PosixReactorBase):
     _asyncClosed = False
     _log = Logger()
 
-    def __init__(self, eventloop=None):
+    def __init__(self, eventloop: Optional[SelectorEventLoop] = None):
 
         if eventloop is None:
             eventloop = get_event_loop()
+            # On Python 3.8+, asyncio.get_event_loop() on
+            # Windows was changed to return a ProactorEventLoop
+            if not isinstance(eventloop, SelectorEventLoop):
+                eventloop = SelectorEventLoop()
```

Comment:
I see that in Tornodo,  the recommended way to use the selector is by calling 

```
asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```
https://docs.python.org/3/library/asyncio-policy.html#asyncio-policies

--------

If in Twisted we create a new eventloop, Twisted will run in its own loop and will not share events with other async libs

Reply:
I've updated the patch, and implemented this.